CLI.md

```diff
@@ -0,0 +1,172 @@
+# AR.IO CLI
+
+The AR.IO CLI is a Node.js command line interface for interacting with the AR.IO network. It allows you to perform various actions such as transferring ARIO, delegating stake, buying records, and more. The CLI is designed to be easy to use and provides detailed help information for each command. This document provides an overview of the CLI and its usage.
+
+<!-- toc -->
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Options](#options)
+- [Commands](#commands)
+- [Development](#development)
+
+<!-- tocstop -->
+
+## Installation
+
+Ensure that you have Node.js installed on your system before installing the CLI. You can check if Node.js is installed by running:
+
+```bash
+node --version
+```
+
+The AR.IO CLI can be installed using npm or yarn. To install the CLI, run one the following commands:
+
+```bash
+npm install -g @ar.io/sdk
+```
+
+or
+
+```bash
+yarn global add @ar.io/sdk --ignore-engines
+```
+
+After installing the CLI, you can run the `ar.io --version` command to verify that the CLI was installed successfully.
+
+```bash
+ar.io --version
+```
+
+## Usage
+
+The AR.IO CLI provides a set of commands that allow you to interact with the AR.IO network. To see a list of available commands, run the following command:
+
+```bash
+ar.io --help
+```
+
+This will display a list of available commands along with a brief description of each command. You can also get detailed help information for a specific command by running:
+
+```bash
+ar.io help <command>
+```
+
+## Options
+
+For each command the AR.IO CLI provides a set of options that allow you to customize the behavior of the CLI. Some of the common options include:
+
+- `--dev`: Run against the AR.IO devnet process
+- `--debug`: Enable debug log output
+- `--skip-confirmation`: Skip confirmation prompts
+- `--ario-process-id <arioProcessId>`: Run against a custom AR.IO process id
+- `--cu-url <cuUrl>`: The URL for a custom compute unit
+- `-w, --wallet-file <walletFilePath>`: The file path to the wallet to use for the interaction
+- `--private-key <key>`: Private key to use with the action
```

Comment:
is this json stringified?

Reply:
yes, it take a full JWK as stringified JSON -- also meant for just a ETH privateKey as a string

Reply:
noted that they were stiringified here: [8ff3f23](https://github.com/ar-io/ar-io-sdk/pull/334/commits/8ff3f234660db477f148c3df68e1d51820d0ec25)